src/50_Samples_%26_Templates/Product.html

```diff
@@ -448,33 +448,33 @@ <h2 class="m1">Apple</h2>
   <amp-state id="product">
     <script type="application/json">
     {
-       "green":{
-          "sizes":{
-             "S":"$5.99",
-             "M":"$5.99",
-             "L":"unavailable"
-          },
-          "defaultSize":"S"
-       },
-       "golden":{
-          "sizes":{
-             "S":"$9.99",
-             "M":"unavailable",
-             "L":"$9.99"
-          },
-          "defaultSize":"L"
-       },
-       "red":{
-          "sizes":{
-             "S":"$7.99",
-             "M":"$7.99",
-             "L":"$7.99"
-          },
-          "defaultSize":"M"
-       }
-     }
-  </script>
-</amp-state>
+      "green": {
+        "sizes": {
+          "S": "$5.99",
+          "M": "$5.99",
+          "L": null
```

Comment:
I think 'unavailable' (as it was before) makes a lot of more sense for someone reading this configuration file.